proto/warden/warden/v1beta3/signature.proto

```diff
@@ -27,6 +27,15 @@ enum SignRequestStatus {
   SIGN_REQUEST_STATUS_REJECTED = 3;
 }
 
+// BroadcastType specifies how the transaction should be broadcasted.
+enum BroadcastType {
+  // The signature should be broadcasted manually by the caller.
+  BROADCAST_TYPE_DISABLED = 0;
```

Comment:
nit: What if we start this enum with 1? And define 0 as `BROADCAST_TYPE_UNSPECIFIED`? I think it's pretty much everywhere like this

Reply:
please read resolved comments above, one of the first from @Pitasi